.github/workflows/installer.yml

```diff
@@ -18,6 +18,15 @@ on: [push, pull_request, workflow_dispatch]
 jobs:
   checktree:
     runs-on: ubuntu-20.04
+
+    strategy:
+      fail-fast: false
+      matrix:  # just a vector in this case
+        make_env: [
+          IPC_VERSION=,  # default version
+          IPC_VERSION=IPC4 UNSIGNED_list= SIGNED_list='cnl icl jsl tgl tgl-h',
```

Comment:
we only support tgl & tgl-h now. And we don't have plan to support old platform with ipc4

Reply:
I know but it compiles now so it provides some poor man's compilation coverage for totally free. Compilation is not official support, it's just mitigation for #5364. Support = validation. As stated in the commit message, it can be removed later with a one-line change if this ever becomes a problem.

Reply:
@RanderWang the main thing is we get full GCC build support for IPC4 code as we are missing this today.

Reply:
thanks